wgsl/index.bs

```diff
@@ -13837,10 +14029,8 @@ fn num_point_lights() -> u32 {
 
     [=Component-wise=] when `T` is a vector.
   <tr>
-    <td>
-    <td>
-
-Note: The result is not mathematically meaningful when `abs(e)` &gt; 1.
+    <td>Scalar [=domain=]
+    <td>Interval [&minus;1, 1]
```

Comment:
Consider dropping interval generally.

Reply:
For cases like `(-inf, inf)` it would look like a two-element set.

Reply:
no change made. see below for why.